src/Lykke.Service.ClientAssetRule/appsettings.json

```diff
@@ -20,7 +20,7 @@
     },
     "ThrottlingLimitSeconds": 60
   },
-  "AssetsServiceClient": {
+  "Assets": {
```

Comment:
Doesn't meet the requirements, why change it from AssetsServiceClient? 
https://lykkex.atlassian.net/wiki/spaces/LKEWALLET/pages/108658780/Code+review+requirements